install.ps1

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env pwsh
+# Copyright 2018 the Deno authors. All rights reserved. MIT license.
+# TODO(everyone): Keep this script simple and easily auditable.
+
+$ErrorActionPreference = 'Stop'
+
+if ($v) {
+  $Version = "v${v}"
+}
+if ($args.Length -eq 1) {
+  $Version = $args.Get(0)
+}
+
+$NixpacksInstall = $env:NIXPACKS_INSTALL
+$BinDir = if ($NixpacksInstall) {
+  "$NixpacksInstall\bin"
+} else {
+  "$Home\.nixpacks\bin"
+}
+
+$NixpacksZip = "$BinDir\nixpacks.zip"
+$NixpacksExe = "$BinDir\nixpacks.exe"
+$Target = 'x86_64-pc-windows-msvc'
```

Comment:
This should also check for `aarch64` and `i686` architectures.

Reply:
Wasn't exactly sure how to match architecture + I was basing this off of Deno's, which doesn't build for those targets.

Reply:
I'll open a pr to your fork.

Reply:
thank you!